R/to_alias.R

```diff
@@ -245,6 +245,8 @@ from_name_to_alias <- function() {
     "sanayi ve ticaret",  "sanayi ticaret",
     "shipping",             "shp",
     "sirketi",             "sti",
+    "s.l.",                "sl",
```

Comment:
Ah, this is from [the other PR ](https://github.com/2DegreesInvesting/tilt.company.match/pull/6).

I do not know if it is possible to separate those commits since the scripts modified are the same ? @maurolepore